zilliqa/src/consensus.rs

```diff
@@ -3205,15 +3205,9 @@ impl Consensus {
                 "view cannot be forced into lower view than current"
             ));
         }
-        match timeout_at.parse::<DateTime>() {
-            Ok(datetime) => self.force_view = Some((view, datetime)),
-            Err(_) => {
-                return Err(anyhow!(
-                    "timeout date must be in format 2001-01-02T12:13:14Z"
-                ));
-            }
-        };
-        self.set_view(view)?;
+        let timeout_at = timeout_at.parse()?;
+        self.force_view = Some((view, timeout_at));
+        self.set_view(view, false)?;
```

Comment:
I added this match for the error response which gives the ISO format required by `DateTime`, as it is otherwise not documented anywhere and fails if eg milliseconds are included in the timestamp